libcloud/compute/drivers/openstack.py

```diff
@@ -1989,11 +1989,7 @@ def _to_node(self, api_node):
         # state
         image = api_node.get('image', None)
         image_id = image.get('id', None) if image else None
-
-        if api_node.get("config_drive", "false").lower() == "true":
-            config_drive = True
-        else:
-            config_drive = False
+        config_drive = api_node.get("config_drive", "false")
```

Comment:
Good catch, but the value should still be a boolean.

If the API now returns a boolean, we can just do `config_drive = api_node.get("config_drive", False)`